StatePrinter.Tests/FieldHarvesters/AnonymousHarvesterTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2014-2015 Kasper B. Graversen
+// 
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using NUnit.Framework;
+using StatePrinting.Configurations;
+using StatePrinting.FieldHarvesters;
+
+namespace StatePrinting.Tests.FieldHarvesters
+{
+  using System.Collections.Generic;
+
+  /// <summary>
+    /// Shows how the <see cref="AnonymousHarvester"/> can be used
+    /// </summary>
+    [TestFixture]
+    class AnonymousHarvesterTest
+    {
+        class B
+        {
+            public int Age;
+        }
+
+        public void SpecializedClassHandler()
+        {
+            var sut = CreatePrinter();
+            var expected = @"new B()
+{
+ Age = ""Its age is 1""
+}";
+            var actual = sut.PrintObject(new B { Age = 1 });
```

Comment:
regarding the rebase,  i did spend time wondering and commenting on the files.. only to see them deleted after making the comments ;)